pages/src/App.tsx

```diff
@@ -0,0 +1,36 @@
+import { Tabs } from "@massalabs/react-ui-kit";
```

Comment:
this is not the ```App```and calling it that way leads to confusion... its more like a way to view the different pages so I would name it: ```DisplayPages.tsx```

Reply:
it should also be moved to a ```components``` / ```utils```folder to put it